src/NServiceBus.Core/Transports/Msmq/MessagePump.cs

```diff
@@ -179,7 +184,17 @@ public void Dispose()
                         {
                             concurrencyLimiter.Release();
                         }
-                    }, cancellationToken, TaskCreationOptions.AttachedToParent,  TaskScheduler.Default);
+                    }, cancellationToken);
+
+                    task.ContinueWith(t =>
```

Comment:
That is well documented knowledge. Sure we need this?

-- 
*#NSBcon 2015 • The "All About NServiceBus" conference is back *
*• Learn more <http://go.particular.net/l/27442/2015-08-21/5d4wzp> →*